module/MOD-INF/controller.js

```diff
@@ -42,7 +42,8 @@ function init() {
     "index/styles",
     module,
     [
-      "styles/files-importing-controller.less"
+      "styles/files-importing-controller.less",
+      "styles/theme.less"
```

Comment:
It's curious we are still using `.less` files, as OpenRefine has migrated to using `.css` a while ago. It could be good to migrate to that here too (perhaps you took example from an older extension?
It should also ideally not be necessary to copy the theme file, but it can well be that it's currently needed. I have opened an issue for this: https://github.com/OpenRefine/sample-extension/issues/30.